industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
> `add_compile_options` is to `target_compile_options` as `include_directories` is to `target_include_directories`.

I'm not sure I follow this. Being more specific is always a good thing in my opinion.

> If I understand it correctly, mixing options between targets might lead to ABI incompabilities, so the per-directory/package directive should be less error-prone.

We wouldn't be mixing options, as `target_compile_options(..)` would be used with all targets in the package. It would also seem to support the [transitive](https://cmake.org/cmake/help/v3.0/manual/cmake-buildsystem.7.html#transitive-usage-requirements) declaration of flags, which seems beneficial to me. I can't find docs on `add_compile_options(..)` stating that it supports that.